

# nav
nav_factor_linear: 0.7
nav_factor_angular: 0.4

# speech
speech_phrases: [
  "hola amiga",
  "hola isabel",
   "yo soy bender, un robot de servicio",
  "ten'ia muchas ganas de conocerte",
  "muchas gracias por ayudarme para viajar a china",
  "eres muy simp'atica",
  "tengo mucha sed, me dar'ias un poquito de aceite?"
]
